fp-proj/src/app/Modules/products/products-list/products-list.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { Product } from '../../../models/Product'
+
+@Component({
+  selector: 'app-products-list',
```

Comment:
fajnie zeby komponenty w aplikacji mialy jakis customowy prefix a nie "app-,,,"
mozna to zmienic w angular.json (i trzeba tez popoprawiac istniejace + zmienic regule w tslint.json)